### PR TITLE
Daily (7/6) GHSA sync script run

### PR DESCRIPTION
**Pull request by jasnow**
Daily GHSA sync script run - redo/more manual checks

### CODE REVIEW

File: gems/jquery-rails/CVE-2015-9251.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+gem: jquery-rails
+framework: rails
+cve: 2015-9251
+ghsa: rmxg-73gg-4p98
+url: https://github.com/rails/jquery-rails/blob/master/CHANGELOG.md#420
```
Comment (postmodern) [inline gems/jquery-rails/CVE-2015-9251.yml:6]: This github URL should be locked to a version tag. ChangeLogs always grow downwards with new releases added to the top.

File: gems/nokogiri/CVE-2021-3537.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+gem: nokogiri
+cve: 2021-3537
+ghsa: 286v-pcf5-25rc
+url: https://nokogiri.org/CHANGELOG.html#security_8
```
Comment (postmodern) [inline gems/nokogiri/CVE-2021-3537.yml:5]: Use the permalink: https://nokogiri.org/CHANGELOG.html#1114-2021-05-14

File: gems/nokogiri/CVE-2021-3518.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+gem: nokogiri
+cve: 2021-3518
+ghsa: v4f8-2847-rwm7
+url: https://nokogiri.org/CHANGELOG.html#security_8
```
Comment (postmodern) [inline gems/nokogiri/CVE-2021-3518.yml:5]: Use the permalink: https://nokogiri.org/CHANGELOG.html#1114-2021-05-14

**Review by postmodern — CHANGES_REQUESTED** [review]
Minor changes needed to the primary URLs so they are static and do not shift.

File: gems/jquery-rails/CVE-2015-9251.yml (hunk 1 above)
Comment (jasnow) [reply gems/jquery-rails/CVE-2015-9251.yml:6]: Changed to https://github.com/rails/jquery-rails/releases/tag/v4.2.0

File: gems/nokogiri/CVE-2021-3537.yml (hunk 2 above)
Comment (jasnow) [reply gems/nokogiri/CVE-2021-3537.yml:5]: Done

File: gems/nokogiri/CVE-2021-3518.yml (hunk 3 above)
Comment (jasnow) [reply gems/nokogiri/CVE-2021-3518.yml:5]: Done

**Comment by jasnow** [comment]
Did the requested changes.

**Review by postmodern — APPROVED** [review]

**Pull request merged**